server/webapp/WEB-INF/rails/webpack/views/components/tab/index.scss

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+$tab_header_border_color: #ccc;
```

Comment:
Can we use something from `_variables.scss` and `theme.scss` ? 
Introducing different backgrounds and borders per page might lead to application inconsistency.

Reply:
same as above